### PR TITLE
WIP. Ccache

### PR DESCRIPTION
**Pull request by stefoss23**
In order to a void recompilation of unchanged source files, setup.py will take advantage of ccache.

### CODE REVIEW

**Comment by stefoss23** [comment]
jenkins build this please

File: python/setup.py [hunk 1]
```diff
@@ -7,6 +7,13 @@
 
 import glob
 import os
+import subprocess
+
+try:
+  subprocess.call(['ccache', '--version'])
+  os.environ['CC'] = 'ccache g++'
```
Comment (joakim-hove) [inline python/setup.py]: People might compile with other c++ compilers - e.g. `clang` - just add an another `subprocess.call(["cc", "--version"])` and then `os.environ['CC'] = 'ccache c++'`

**Comment by stefoss23** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by atgeirr** [comment]
This was merged while marked WIP, I assume that was ok and the name was misleading at the time of merging?

**Comment by joakim-hove** [comment]
> I assume that was ok and the name was misleading at the time of merging?

Yes; but point taken anyway.

**Pull request merged**